boringssl-static/pom.xml

```diff
@@ -375,9 +375,9 @@
                   <instructions>
                     <Export-Package>io.netty.internal.tcnative.*</Export-Package>
                     <Bundle-NativeCode>
-                      META-INF/native/libnetty_tcnative_osx_x86_64.jnilib;osname=macos;osname=macosx;processor=x86_64,
-                      META-INF/native/libnetty_tcnative_linux_x86_64.so;osname=linux;processor=x86_64,
-                      META-INF/native/netty_tcnative_windows_x86_64.dll;osname=win32;processor=x86_64
+                      META-INF/native/libnetty_tcnative_osx_x86_64.jnilib;osname=macos;osname=macosx;processor=${os.detected.arch},
+                      META-INF/native/libnetty_tcnative_linux_x86_64.so;osname=linux;processor=${os.detected.arch},
```

Comment:
Is the library name still x86_64.so, even if it my not be 64 bit?

Reply:
good catch! Let me fix this 👍